### DISCUSSION TITLE
First perspective of Jukkasjärvi Church Sample Set is released

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#2064] [General]
After a lot of work the first perspective, actually the AB position that was recorced with a pair of Oktava MK-012 microphones, is now released. More information is available at [http://familjenpalo.se/vpo/jukkasjarvi-church-organ-sample-set/](http://familjenpalo.se/vpo/jukkasjarvi-church-organ-sample-set/) and the download link can either be found there or way down in the table at [http://familjenpalo.se/vpo/download/](http://familjenpalo.se/vpo/download/) where also the md5sum and sha1sum can be found to check the download integrity.

Just a word of warning for those expecting a cathedral acoustics: the Jukkasjärvi church is a very dry room! There's no more than about a second of reverb tail, so if more reverb is wanted it must be added artificially by IR or some other means. However, the audio quality is very detailed and the instrument in itself is quite charming in its own right. Enjoy!

**Comment by mutti-wilson** [comment]
I'll have to check this out. I was at a summer service in Jukkasjärvi in the summer of 2011.

**Reply by larspalo** [reply]
@mutti-wilson Wow! Then you must have heard the organ when it was reasonably new. Unfortunately the tremulant was out of service when I recorded the samples so it's only a synthetic version in the sample set. Even so, I think the organ is very charming. A testimony to its uniqueness is that the renowned Swedish organist Gunnar Idenstam visits Jukkasjärvi and gives concerts there on a somewhat regular basis.

The extended version has a lower octave for the second manual that can be used by the octave coupler(s) and also a separate Basun 16' stop in the pedal that's an extension of the Trumpet 8'. This makes the sound surprisingly full for such a relatively small disposition!

**Comment by Spillregal16** [comment]
Hello Lars Palo, I am a user of your Sample Sets and find your work very nice. I wanted to notify you that when you open the last Sample Set of Jukkasjärvi Church it shows "Error: Missing value in section 'Rank001' for entry 'Percussive'. 

p.s I think it's great that they're one of the few still recording free Sample Sets  for Grand Orgue 👏👍

**Reply by larspalo** [reply]
> it shows "Error: Missing value in section 'Rank001' for entry 'Percussive'

Yes, I know about it, it's because the 'Percussive' key has changed to be an optional entry in more recent versions of GrandOrgue. A simple fix for any user is just to upgrade their GO and the error will be gone.

**Reply by Spillregal16** [reply]
Thanks for your answer

**Comment by larspalo** [comment]
The second perspective of this sample set, the near coincident microphone pair, is now published. This perspective was recorded using my trusty Studio Projects B1 condenser microphones.

At the very bottom of http://familjenpalo.se/vpo/jukkasjarvi-church-organ-sample-set/ a download link can be found as well as way down in the table at http://familjenpalo.se/vpo/download/ below which also the checksums are published that can be used to verify the download integrity.

While the difference for some might appear to be subtle compared to the AB version, I personally really like the crispness and clear directional information that this perspective have. Enjoy!
